lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -24,9 +67,13 @@ previewers.file_maker = function(filepath, bufnr, opts)
     filepath = vim.fn.expand(filepath)
     local stat = vim.loop.fs_stat(filepath) or {}
     if stat.type == 'directory' then
-      pscan.ls_async(filepath, { hidden = true, on_exit = vim.schedule_wrap(function(data)
-        vim.api.nvim_buf_set_lines(bufnr, 0, -1, false, data)
-        if opts.callback then opts.callback(bufnr) end
+      pscan.ls_async(filepath, {
+        hidden = true,
+        group_directories_first = true,
```

Comment:
Anyone a issue with this option? Because i like it :)

Reply:
No, I don't have any issue and I like it as well!